### ISSUE TITLE
SonarQube Fails to Boot Due To Permissions Issues

### ISSUE THREAD

**Issue by sherl0cks** [InfoSec812/openshift-sonarqube#12]
In OCP 3.6, deploying the image leads to the following failure, which then leads to a crash loop. This failure is not consistent between deployments. It works sometimes and fails others.

```

  | + set -e
-- | --
  | ++ ls -A /opt/sonarqube/data
  | + [[ -n lost+found
  | plugins ]]
  | + [[ -d /opt/sonarqube/data/plugins ]]
  | + rm -rf /opt/sonarqube/extensions/plugins
  | rm: cannot remove '/opt/sonarqube/extensions/plugins/README.txt': Permission denied

```

**Comment by InfoSec812** [comment]
Yeah, this is new and not understood as nothing has changed in the Dockerfile to cause this. I have raised this issue with Øystein already.

**Comment by sherl0cks** [comment]
closed via #13
